akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/DDataRememberEntitiesShardStore.scala

```diff
@@ -86,8 +88,8 @@ private[akka] final class DDataRememberEntitiesShardStore(
   implicit val selfUniqueAddress: SelfUniqueAddress = SelfUniqueAddress(node.selfUniqueAddress)
 
   private val readMajority = ReadMajority(settings.tuningParameters.waitingForStateTimeout, majorityMinCap)
-  // Note that the timeout is actually updatingStateTimeout x 3 since we do 3 retries
-  private val writeMajority = WriteMajority(settings.tuningParameters.updatingStateTimeout, majorityMinCap)
+  // Note that the timeout is actually updatingStateTimeout / 4 so that we fit 3 retries and a response in the timeout before the shard sees it as a failure
+  private val writeMajority = WriteMajority(settings.tuningParameters.updatingStateTimeout / 4, majorityMinCap)
```

Comment:
@chbatey not 100% sure this is a good solution, have any better idea ?

Reply:
I think this is fine. No harm in parts of the get all timing out and retrying